www/_posts/2023-05-22-cordova-android-12.0.0.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 12.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 12.0.0`! This is one of Cordova's supported platforms for building Android applications.
+
+* [cordova-android@12.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@12.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add android@12.0.0
+```
+
+* **Increased Minimum & Target SDK**
+
+    This release has increased the minimum supported SDK version to 24 which is Android 7.0. It also has increased the target SDK to 33, Android 13.
+
+* **Tooling and Default Support Bump**
+
+  * Target SDK (`targetSdk`): `33`
+  * SDK Build Tool: `33.0.2`
+  * Kotlin: `1.7.21`
+  * Gradle: `7.6`
+  * Android Gradle Plugin (AGP): `7.4.2`
+  * Google Services Gradle Plugin: `4.3.15`
+  * AndroidX App Compat Library: `1.6.1`
+  * AndroidX WebKit Library: `1.6.0`
+  * AndroidX SplashScreen Core Library: `1.0.0`
+
+* **Monochrome Support**
+
+    Android 13 has added Themed Icons support which is also known as Monochrome. This release has introduced support for Themed icons.
+
+* **Node Support**
+
+    We have dropped support for Node 14.x and increase the minimum Node requirement to greater than or equal to 16.13.0.
+
+Please report any issues you find at [issues.cordova.io](http://issues.cordova.io/)!
```

Comment:
This link is actually dead, I think we should just link to https://github.com/apache/cordova-android/issues instead.

That's what I did for the browser release announcement anyway.